modules/all-modules/pom.xml

```diff
@@ -15,6 +15,12 @@
     </description>
 
     <dependencies>
+        <!-- TODO(gfernandes) added due to HTTP extension needs to access the Sockets extension API, remove this once MULE-9867 is fixed  -->
```

Comment:
Same as previous comment, in order to work http needs to have sockets registered as extension plugin plus its jar available in container as mule module.